docs/docs/workflows/verify-cluster.md

```diff
@@ -15,6 +15,15 @@ This command performs the following steps:
 2. Verify the signature of the measurements. This will use Edgeless Systems' [public key](https://edgeless.systems/es.pub).
 3. Write measurements into configuration file.
 
+After the command succeeded you will see more entries under the `measurements` key than there are indices in the `enforcedMeasurements` key.
+This is because only a subset of the available measurements can be verified and reproduced.
+These non-reproducible values typically measure parts of the boot process that are controlled by the CSP.
+You can find a list with descriptions in the [runtime measurements](../architecture/attestation.md#runtime-measurements) section.
+
+When a new node tries to join the cluster, the [join service](../architecture/components.md#joinservice) will compare each [PCR](../architecture/attestation.md#platform-configuration-register-pcr) individually.
+For mismatching PCRs that are set under the `measurements` key a warning will be emitted.
+For mismatching PCRs that are additionally set under `enforcedMeasurements` an error will be emitted and the new node won't be able to join the cluster.
```

Comment:
```suggestion
During attestation, the validating side will compare each [measurement](../architecture/attestation.md##runtime-measurement) reported by the issuing side individually.
For mismatching measurements that are set under the `measurements` key a warning will be emitted.
For mismatching measurements that are additionally set under `enforcedMeasurements` an error will be emitted and attestation fails.
```
* PCRs should be referred to as measurements in user facing documentation

* Since this page talks about the `verify` command, I think it would be better to talk about attestation, instead of explaining how this applies to the join-service.